workflow/search-filter.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Search & Filter
+---
+
+The Search & Filter features serve as the jumping-off point in your data exploration process within Middleware. Data is made available across all contexts within Middleware, allowing you to connect all data points collected.
+
+The Filter Bar and Search Fields are two of the simplest methods for sifting through your data. They are both dynamically generated based on the applications, integrations, and infrastructure you have instrumented within Middleware. These features are inclusive (i.e. union set operations) and will display data that satisfies any of the selected filters.
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaTFkdDIwa295MWlhM2liNTk1ZmM0ZHI2NnZ3aTA0c2wzdnVoOHgxbyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/awGRnLE9t3NZG1sTde/giphy.gif" width="750" height="750"/>
+
+# Using the Searchbar
+
+### Attribute to Value Matching
+
+The searchbar allows you to dynamically select data that has been generated within the page you are searching through. Your instrumented applications contain a default set of attributes that are collected and stored within Middleware that correspond to dynamically generated values. Begin your search by clicking into the searchbar, selecting your desired attribute and then choosing a corresponding value that appears shortly after selecting said attribute:
+
+----gif here-----
+
+### Logical Operators
+
+Below is a table containing the available logical operators you can use within the searchbar:
+
+| __Operator__ | __Description__ |
+| :----------- | :-------------- |
+| `AND`        | Intersection: Both terms must be present within the selected search |
+| `OR`         | Union: Either term is contained within the selected search |
+
+### Regex Operators
+
+Regex operators allow you to identify specific patterns of strings within your instrumented application data.
```

Comment:
Let's add what type of regex we support. There are many different regex standards. I _think_ Ed knows which standard we follow.